.yarn/patches/source-map-js-npm-1.0.2-ee4f9f9b30.patch

```diff
@@ -0,0 +1,63 @@
+diff --git a/lib/quick-sort.js b/lib/quick-sort.js
+index 23f9eda57f7383b5291361e4593cf2fdef5409ba..016785171b4d69d756ac795763abc6cf2c9d1b31 100644
+--- a/lib/quick-sort.js
```

Comment:
why is this patch necessary ? is it for performance reasons that you don't to clone and cache sort instances ?

Reply:
The original code uses evaluation (in L41 ```let templateFn = new Function(`return ${template}`)();```) and it is not allowed in devtools :( So I found the related [commit](https://github.com/7rulnik/source-map-js/pull/2/files) in source-map-js and revert this commit via a patch.

Actually the commit is about perf improvement, so according to that the patched version is less performant. But it's better than having nothing.... I'm really surprised how few libraries are doing source map parsing.

Reply:
does react-devtools use this library ?

Reply:
It does not, it parses sourcemap on its own 🙈

Reply:
here it is: https://github.com/facebook/react/blob/b66936ece7d9ad41a33e077933c9af0bda8bff87/packages/react-devtools-shared/src/hooks/SourceMapConsumer.js 
It's not very long but I would prefer the library

Reply:
I would trigger an issue to the library, see if they can release a version where your alternative could exist

Reply:
https://github.com/7rulnik/source-map-js/issues/15